src/main/java/carpet/script/api/Scoreboards.java

```diff
@@ -115,6 +115,43 @@ public static void apply(Expression expression)
         // objective_add('counter')
 
         expression.addLazyFunction("scoreboard_add", -1, (c, t, lv)->
+        {
```

Comment:
why you don't just use Expression.alias for that?

Reply:
cos.. I didnt know it existed...